tests/Squidex.Domain.Apps.Core.Tests/Model/Contents/WorkflowJsonTests.cs

```diff
@@ -19,8 +24,33 @@ public void Should_serialize_and_deserialize()
             var workflow = Workflow.Default;
 
             var serialized = workflow.SerializeAndDeserialize();
+            var test = workflow.GetTransitions(Status.Archived);
 
             serialized.Should().BeEquivalentTo(workflow);
         }
+
+        [Fact]
+        public void Should_verify_roles_mapping_in_workflow_transition()
+        {
+            var source = new
```

Comment:
I prefer to have this single lined when there are only a few properties.

Reply:
I've updated it.